### ISSUE TITLE
Flaxbeard's Steam Power alloys?

### ISSUE THREAD

**Issue by Qohelethh** [joshiejack/ModTweaker#85]
There doesn't seem to be support for adding recipes to make alloys with the crucible. Is this the case, or am I doing something wrong? I can add recipes to melt down ingots, but I can't figure out how to create recipes for alloys.

**Comment by Yulife** [comment]
Did you read the [wiki entry](http://minetweaker3.powerofbytes.com/wiki/ModTweaker:Flaxbeard_Steam_Power_Support) for it? ^-^ I only made it recently so you might have missed it!

**Comment by Qohelethh** [comment]
Yeah, maybe I'm just dumb, but I can't figure out how to add more than one input stack to the recipe. 

Specifically, I'm trying to make it so the crucible can make electrum from Thermal Foundation by melting one gold and one silver ingot together to produce two ingots of electrum.

**Comment by Yulife** [comment]
Oh.... I get what you mean now. @jaredlll08
